docs/articles/nunit/running-tests/Console-Command-Line.md

```diff
@@ -34,7 +34,7 @@ Without the V2 driver, only version 3.0 and higher tests may be run.
 | ------ | ----------- |
 |`@FILE` | Specifies the name (or path) of a FILE containing additional command-line arguments to be interpolated at the point where the @FILE expression appears. Each line in the file represents a separate command-line argument.|
 |`--test=FULLNAMES` | Comma-separated list of FULLNAMES of tests to run or explore. This option may be repeated. Note that this option is retained for backward compatibility. The --where option can now be used instead. |
-|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. |
+|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. May also include comment lines, indicated by `#` in the first column.|
```

Comment:
```suggestion
|`--testlist=FILE` | The name (or path) of a FILE containing a list of tests to run or explore, one per line. May also include comment lines, indicated by `#` in the first column. |
```

Adding a space prior to the pipe. Very very minor. I'll commit this one myself and then merge.